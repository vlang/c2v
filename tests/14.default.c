const int a = 0;

void f() 
{ 
    switch(a)
    {
    default:
    case 0:
    case 1:
        break;
    }

    switch(a)
    {
    default:
    case 0:
        break;
    }

    switch(a)
    {
    case 0:
        break;
    case 1:
        break;
    default:
    break;
    }
    
}