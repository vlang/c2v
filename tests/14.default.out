[translated]
module main

[export: 'a']
const (
	a = 0
)

fn f() {
	match a {
		0, 1 {}
		else {}
	}
	match a {
		0 { // case comp body kind=BreakStmt is_enum=true
		}
		else {}
	}
	match a {
		0 { // case comp body kind=BreakStmt is_enum=true
		}
		1 { // case comp body kind=BreakStmt is_enum=true
		}
		else {}
	}
}
